src/smart-components/OpenChannel/components/OpenChannelMessageList/__tests__/useHandleOnScrollCallback.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { useHandleOnScrollCallback, calcScrollBottom } from '../useHandleOnScrollCallback';
+import { renderHook } from '@testing-library/react'
+
+const prepareMockEvent = ({
+  scrollTop = 0,
+  scrollHeight = 0,
+  clientHeight = 0,
+}) => {
+  return {
+    target: {
+      scrollTop: scrollTop || 0,
+      scrollHeight: scrollHeight || 0,
+      clientHeight: clientHeight || 0,
```

Comment:
```suggestion
      scrollTop,
      scrollHeight,
      clientHeight,
```
I guess this should fine cause we're already setting their default values at the above?

Reply:
yeah, its cool, will add ~